lib/aptible/cli/subcommands/operation.rb

```diff
@@ -16,6 +16,24 @@ def self.included(thor)
               CLI.logger.info m
               o.update!(cancelled: true)
             end
+
+            desc 'operation:connect OPERATION_ID',
+                 'Follow logs of a running operation'
+            define_method 'operation:connect' do |operation_id|
+              o = Aptible::Api::Operation.find(operation_id, token: fetch_token)
+              raise "Operation ##{operation_id} not found" if o.nil?
+
+              if %w(failed succeeded).include? o.status
+                raise Thor::Error, "This operation has already #{o.status}. " \
+                                  'Only currently running operations are ' \
+                                  'supported by this command at this time.'
+              end
+
+              m = "Streaming logs for #{prettify_operation(o)}..."
```

Comment:
Nitpick - May want to reuse the verb connecting here or some other consistent verb/action

Reply:
I think "Streaming" is appropriate here.